code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1904,6 +1904,56 @@
 			changeling.chem_charges = max(changeling.chem_charges-2, 0)
 	return ..()
 
+/datum/reagent/concentrated_bz
+	name = "Concentrated BZ"
+	description = "A hyperconcentrated liquid form of BZ gas, known to cause an extremely adverse reaction to changelings. Also causes minor brain damage."
+	color = "#FAFF00"
+	taste_description = "acrid cinnamon"
+	random_unrestricted = FALSE
+
+/datum/reagent/concentrated_bz/on_mob_metabolize(mob/living/L)
+	..()
+	ADD_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
+
+/datum/reagent/concentrated_bz/on_mob_end_metabolize(mob/living/L)
+	..()
+	REMOVE_TRAIT(L, CHANGELING_HIVEMIND_MUTE, type)
+
+/datum/reagent/concentrated_bz/on_mob_life(mob/living/L)
+	if(L.mind)
+		var/datum/antagonist/changeling/changeling = L.mind.has_antag_datum(/datum/antagonist/changeling)
+		if(changeling)
+			if(prob(30))
+				L.adjustStaminaLoss(3,5)
+				L.emote("scream")
+				to_chat(L, "<font size=3 color=red><b>The chemicals burn our senses!</b></font>")
+				L.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2, 50)
+		else
+			L.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5, 200)
+			if(prob(30))
+				to_chat(L, "<font size=3 color=red><b>Your head feels like it's on fire!</b></font>")
+	..()
+
+/datum/reagent/fake_cbz
+	name = "Concentrated BZ"
+	description = "A hyperconcentrated liquid form of BZ gas, known to cause an extremely adverse reaction to changelings. Also causes minor brain damage."
```

Comment:
Mentions nothing of its effects on non-lings.

Reply:
> Also causes minor brain damage.